docs/guides/web-ui/dashboards.md

```diff
@@ -1,111 +1,138 @@
+You can add more panel groups to better organize your dashboard. This is useful for grouping related visualizations, effectively allowing you to have multiple views within a single dashboard.
 
-![Logfire Dashboard chart query](../../images/guide/browser-dashboard-chart-sql-query.png)
+To add a new group, click the **Panel Group** button in the top right. You can name the group and set whether it should be expanded or collapsed by default when the dashboard loads.
 
-### Chart Preview and configuration
+To add a new visualization, you add a panel to a group. Click the **Panel** button in the top right. Inside each panel, you'll configure a chart and the SQL query that powers it.
 
-Based on your need and query, you need to configure the chart to visualize and display your data:
+You can rearrange and resize panels by dragging and dropping them after clicking the **Edit layout** button.
 
-#### Time Series Chart
+### Chart Types
 
-A time series chart displays data points over a specific time period.
+Logfire supports these chart types:
 
-#### Pie Chart
+| Chart Type  | Query Type         | Description                          |
+| ----------- | ------------------ | ------------------------------------ |
+| Time Series | TimeSeriesQuery    | Data points over time                |
+| Gauge       | TimeSeriesQuery    | Shows current value as a gauge       |
+| Table       | NonTimeSeriesQuery | Tabular data with rows and columns   |
+| Bar Chart   | NonTimeSeriesQuery | Comparisons across categories        |
+| Pie Chart   | NonTimeSeriesQuery | Data as slices of a circle |
+| Values      | NonTimeSeriesQuery | Values displayed as tiles |
 
-A pie chart represents data as slices of a circle, where each slice represents a category or value.
+!!! note
+    Queries with `TimeSeriesQuery` as the **Query Type** must return a timestamp column.
 
-#### Table
+To configure a chart:
 
-A table displays data in rows and columns, allowing you to present tabular data.
+1. Choose the chart type.
+2. Write your SQL query.
+3. Customize the formatting, labels, and appearance.
 
-#### Values
+---
 
-A values chart displays a single value or multiple values as a card or panel.
+## Writing Queries
 
-#### Categories
+Logfire uses SQL to define dashboard queries.
 
-A categories chart represents data as categories or groups, allowing you to compare different groups.
+If you're unsure which tables or columns are available, refer to the [records schema](explore.md#records-schema) and [metrics schema](explore.md#metrics-schema).
 
-## Tips and Tricks
+### Variable Usage
 
-### Enhanced Viewing with Synchronized Tooltips and Zoom
+You can reference dashboard variables in SQL queries using the `$variable` syntax:
 
-For dashboards containing multiple time-series charts, consider enabling "Sync Tooltip and Zoom." This powerful feature provides a more cohesive viewing experience:
+```sql
+SELECT * FROM records WHERE service = $service_name
```

Comment:
```suggestion
SELECT * FROM records WHERE service_name = $service_name
```

There's no `service` column.

But also this makes it sound like there's some built in `$service_name` variable. You're explaining how to use variables in queries before how to define them. Users won't know what you're saying.

This query forces the user to fill in a service name, it doesn't allow seeing the full set of records unfiltered.